src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculate = {
+    add(num) {
+      result += num;
+    },
+    subtract(num) {
+      result -= num;
+    },
+    multiply(num) {
+      result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        result /= num;
+      } else {
+        throw new Error('You can divide on zero');
```

Comment:
The error message in the divide method should be corrected to 'You cannot divide by zero' instead of 'You can divide on zero'.